_data/Raghav-byte/Raghav-byte.md

```diff
@@ -0,0 +1,5 @@
+name: Raghav Shukla
+institution: Shri Shankracharya Institute of Prof. Management and Tech.
+profile_pic: Raghav-byte.jpg
+quote: "Why Fall In Love When You Can Fall Asleep"
```

Comment:
Please follow the given instruction by removing the quotes to maintain the same format

Reply:
i did the needful , please check again !
Thanks

Reply:
Seems like you haven't removed the quotes yet

Reply:
> Seems like you haven't removed the quotes yet

Let me know when you resolve this

Reply:
i did . Thanks